packages/tools/devtools/devtools-view/src/components/ContainerSummaryView.tsx

```diff
@@ -81,8 +79,21 @@ interface Item {
 	value: string;
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-function DataRow(label: string, id: string | undefined, columnProps: any): React.ReactElement {
+/**
+ * Displays a row with basic stats about the Container.
+ *
+ * @param label - Row label text.
+ * @param infoTooltipText - Tooltip text to display via an info badge.
+ * @param value - The value text associated with the label.
+ * @param columnProps - Column props consumed by FluentUI.
+ */
+function DataRow(
+	label: React.ReactElement | string,
+	infoTooltipText: React.ReactElement | string,
```

Comment:
Might need support for `undefined` too? Or at least skip the wrapping if a blank string is passed in?

Reply:
That's fair. My hope is that we'll have info badges for everything in the table here, but just in case. Will update.